CliWrap/Internal/ProcessEx.cs

```diff
@@ -51,12 +51,19 @@ public void Start()
             StdErr = _nativeProcess.StandardError.BaseStream;
         }
 
-        public bool TryKill()
+        public bool TryKill(bool gracefully = false)
         {
             try
             {
                 _nativeProcess.EnableRaisingEvents = false;
-                _nativeProcess.Kill(true);
+                if (gracefully)
+                {
+                    Cli.Wrap("taskkill").WithArguments("/PID " + Id).ExecuteAsync();
```

Comment:
This will be a fire and forget async call. If `TryKill()` needs to be sync then it should be `.Result`-ed, otherwise `TryKillI()` should be converted to async and this call `await`-ed.

Reply:
I think we should just use `Process.Start("taskkill", $"/PID {Id}")` instead for a fire-and-forget execution.